src/iris_grib/tests/integration/load_convert/test_sample_file_loads.py

```diff
@@ -118,6 +118,8 @@ def test_reduced_ll(self):
         )
         self.assertCML(cube, _RESULTDIR_PREFIX + ("reduced_ll_grib1.cml",))
 
+    # TODO: fix this - possibly just update the reference cml
+    @pytest.mark.skip(reason="known change in eccodes?")
```

Comment:
Is it possible that your branch is just out of date? I updated the CML in #716

Reply:
It is possible that there is a weird intermittent problem here.  The data signatures (new-style) don't match.
If I comment out the skip, I get ...
```
E   AssertionError: CML do not match: /home/users/patrick.peglar/git/iris_grib/src/iris_grib/tests/results/integration/load_convert/sample_file_loads/reduced_gg_grib1.cml
E   --- Reference
E   +++ Test result
E   @@ -43 +43 @@
E   -    <data checksum="0x514a7763" dtype="float64" shape="(6114,)"/>
E   +    <data checksum="0x6fe0ee2c" dtype="float64" shape="(6114,)"/>
```

Reply:
OK so the branch is up to date; what about the environment you are using for local testing? It needs to be ecCodes version `2.44.0` or higher.